scripts/nemo/canary/test_1b_flash.py

```
@@ -0,0 +1,303 @@
+    features_len = np.array([features.shape[1]], dtype=np.int64)
+
+    enc_states, _, enc_masks = model.run_encoder(features, features_len)
+
+    decoder_input_ids = []
+    decoder_input_ids.append(token2id["<|startofcontext|>"])
+    decoder_input_ids.append(token2id["<|startoftranscript|>"])
+    decoder_input_ids.append(token2id["<|emo:undefined|>"])
+    if args.source_lang in ("en", "es", "de", "fr"):
+        decoder_input_ids.append(token2id[f"<|{args.source_lang}|>"])
+    else:
+        decoder_input_ids.append(token2id[f"<|en|>"])
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)